x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/components/delete_domain_panel.tsx

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { useActions, useValues } from 'kea';
+
+import { EuiButton, EuiSpacer, EuiText } from '@elastic/eui';
+
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+
+import { CrawlerSingleDomainLogic } from '../crawler_single_domain_logic';
+
+export const DeleteDomainPanel: React.FC = ({}) => {
+  const { domain } = useValues(CrawlerSingleDomainLogic);
+  const { deleteDomain } = useActions(CrawlerSingleDomainLogic);
+
+  if (!domain) {
+    return null;
+  }
+
+  return (
+    <>
+      <EuiText size="s">
+        <p>
+          <FormattedMessage
+            id="xpack.enterpriseSearch.appSearch.crawler.deleteDomainPanel.description"
+            defaultMessage="Remove this domain from your crawler. This will also delete all entry points and crawl
+                    rules you have setup. {cannotUndoMessage}."
```

Comment:
Should this be wrapped with i18n?

Reply:
The reason I'm using it is that we can't include JSX/HTML inside of the defaultMessage of an `i18n.translate`

Reply:
Makes sense 👍